app/models/oral_history_content.rb

```diff
@@ -32,6 +32,15 @@ class OralHistoryContent < ApplicationRecord
   include CombinedAudioUploader::Attachment.new(:combined_audio_mp3, store: :combined_audio_derivatives)
   include CombinedAudioUploader::Attachment.new(:combined_audio_webm, store: :combined_audio_derivatives)
 
+  job_status_options = {
+    queued: 'queued',
+    started: 'started',
+    failed: 'failed',
+    done: 'done'
+  }
+
+  enum combined_audio_derivatives_job_status: job_status_options
```

Comment:
No reason to have this as a separate local variable first, that's never re-used. Just put the hash inline, I think it's more readable inline, not making the reader look in two places and put it together in their head.